demo/apiExamples.md

```diff
@@ -8,10 +8,10 @@
 |---------------|---------------|-----------|-------------|--------------------------------------------------|
 | [centralDate](#centralDate) | `centralDate` | `Object`  |             | The date that determines the currently visible month. |
 | [disabled](#disabled)    | `disabled`    | `Boolean` |             | If set, disables the datepicker.                 |
-| [error](#error)       | `error`       | `Boolean` | false       | Sets a persistent error state (e.g. an error state returned from the server). |
 | [maxDate](#maxDate)     | `maxDate`     | `Date`    | "undefined" | Maximum date. All dates after will be disabled.  |
 | [minDate](#minDate)     | `minDate`     | `Date`    | "undefined" | Minimum date. All dates before will be disabled. |
 | [required](#required)    | `required`    | `Boolean` |             | Populates the `required` attribute on the input. Used for client-side validation. |
+| [validity](#validity)    | `validity`    | `String`  | "undefined" | Specifies the `validityState` this element is in. |

```

Comment:
I am having an issue with using `validity` as an enumerable attribute. If the goal is to more closely bind ourselves to the HTML spec, this really should be `valitidystate`. 

Also, we should clearly state in the documentation that we are expecting string values that are consistent with the client-side validation expectations listed here.https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#client-side_validation

While I can see using this to our benefit internally, what is the benefit to the end user to have to pass in a specific state? 

I really like how Shoelace simplified this and handled this with an `invalid`  boolean. https://shoelace.style/components/input#:~:text=false-,invalid,-This%20will%20be